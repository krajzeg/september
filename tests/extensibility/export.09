implicit_export := {
	fourtyTwo := 42
	export(fourtyTwo)
}

implicit_export()
print("The exported value should be 42:", fourtyTwo)

explicit_export := |name| {
	export(42, as: name)
}

explicit_export("theAnswer")
print("And the explicitly exported value should also be 42:", theAnswer)
