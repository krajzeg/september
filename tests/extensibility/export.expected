The exported value should be 42: 42
And the explicitly exported value should also be 42: 42
